packages/altive_lints/lint_test/lints/avoid_hardcoded_japanese_test.dart

```diff
@@ -1,7 +1,6 @@
 // Check the `avoid_hardcoded_japanese` rule.
 //
-// This file ends with the name `_test.dart`,
-// so it should be exempt from the warning.
+// It should exclude warnings for the entire `/test/` directory.
```

Comment:
![badge](https://img.shields.io/badge/review-must-critical)
This file is not in the `/test/` directory.
I think you need to create a `/test/` directory and try it.

Reply:
Since the `/test/` directory does not currently exist, I would like to create a `/test/` directory and deal with it there.

I would like to create the `/test/` directory on top of `/lint_test/`.

Reply:
Instead of creating a `/test/` directory on top of the `/lint_test/` directory, the name was changed from `/lint_test/`  to `/test/`.

The reason is that it feels strange to have a `/lint_test/` directory inside a `/test/` directory, and it seems better to just keep it short and simple as test directory.

Change : https://github.com/altive/altive_lints/pull/70/commits/62d3bcdc710ed810e1145d92a1dfce16c6676c07

Reply:
@naipaka 
I have renamed the common `lint_test` directory to the `test` directory, is this ok?
I thought I should only add the `test` directory for rules that I want to test in the `test` directory, but if renaming all `lint_test` to `test` is ok, then maybe this is ok too🤔

Reply:
In my understanding, there is an issue.

The Dart test framework usually requires a `main` function. However, for lint rule testing, the `main` function is not utilized, which is why I created a separate `lint_test` directory instead of using `test`. (At the time, I believe an error indicating that a main entry point could not be found when running `flutter test` was occurring…)

Currently, `pubspec.yaml` and `analysis_options.yaml` are located within the `test` directory, which also seems unusual.

---

The name `lint_test` might have been misleading. Looking at other repositories that use custom_lint for lint packages, it seems that some check for lints within the `example` directory using `expect_lint: xxx`. Therefore, the following directory structure might be a suitable alternative:

```
altive_lints/
├── lib/
├── example/
│   └── main.dart
│   └── lints
│       └── avoid_consecutive_sliver_to_box_adapter.dart
│   └── test
│       └── xxx.dart
└── pubspec.yaml
```

Reply:
@naipaka 
Thanks for the details!

@boywithdv 
I have a problem with the `test` directory, so please change it to the above configuration.

Reply:
@naipaka 
Thank you for the detailed explanation!

@riscait 
> I have a problem with the test directory, so please change it to the above configuration.

Yes, sir.
We will make the correction again, and if it looks OK, we will submit another review request.